### DISCUSSION TITLE
Add a more realistic/efficient operating mode for the Ferris Wheel.

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21414] [Feature requests]
Presently, the only operating mode for the Ferris Wheel loads one car, then performs the set number of rotations before loading the next car. As there around 16 cars on the Ferris Wheel, this means that any given car of guests will be on the ride for a multiple of 16 rotations, rather than the actual set number.

I'm suggesting here a mode where the Ferris Wheel loads all 16 cars before performing the number of rotations, then unloading/loading them all again before performing those rotations once more.

As far as names for these rotation modes, it might be called something like,
Continuous load rotation (forwards/backwards) [this would be the current operating mode]
Full load rotation (forwards/backwards) [this would be the new operating mode].

**Comment by ocalhoun6** [comment]
Perhaps a half-rotation mode could be considered as well? Where instead of doing a full rotation before loading each guest, it does 1/2 rotation, which means it loads guests twice as fast and each guest is on it for 1/2 as many rotations?

(Overall, this is a pretty good idea. Even just setting a Ferris Wheel to 2 rotations will leave guests thinking the ride is too long and they want to get off of it. New operating modes like this would make changing the number of rotations much more viable to do in an actual park, plus it could make the Ferris Wheel -- traditionally a very poor financial performer -- a *little* bit more of a money-maker.)

------

Maybe the new mode could be called "Sequential Mode"?

**Comment by rondadams** [comment]
I'm guessing it was done this way for ease of handling the guests loading/unloading and tracking how many times each had gone around. Not sure what the mode change would mean as for ease of tracking in the code.

**Comment by cheweytoo** [comment]
I was wondering if someone had already started a discussion about that :) – yes, this leads to completely ridiculous ride durations even with the "one rotation" setting. Additionally, this is most certainly _not_ how ferris wheels operate IRL.

The "number of rotations" setting should actually mean the number of rotations experienced by a guest on the ride.
